pyspecdata/core.py

```diff
@@ -6871,15 +6872,28 @@ def myfilter(x,center = 250e3,sigma = 100e3):
 
 #{{{ fitdata
 class fitdata(nddata):
-    r''' Inherits from an nddata and enables curve fitting through use of a sympy expression. The user creates a fitdata class object from an existing nddata class object, and on this fitdata object can define the func:`functional_form` of the curve it would like to fit to the data of the original nddata. This functional form must be provided as a sympy expression, with one of its variables matching the name of the dimension that the user would like to fit to. The user provides fit coefficients using func:`fit_coeff` and obtains output using func:`fit` and func:`eval`.'''
-    If you haven't dont his before,
+    r'''Inherits from an nddata and enables curve fitting through use of a sympy expression.
+
+    The user creates a fitdata class object from an existing nddata
+    class object, and on this fitdata object can define the
+    func:`functional_form` of the curve it would like to fit to the
+    data of the original nddata.
+    This functional form must be provided as a sympy expression, with
+    one of its variables matching the name of the dimension that the
+    user would like to fit to.
+    The user provides fit coefficients using func:`fit_coeff` and
+    obtains output using func:`fit` and func:`eval`.
+
+    If you haven't done this before,
     create a jupyter notebook (not checked in, just for your own playing around) with:
     ```
     import sympy as s
     s.init_printing()
     ```
-    you can then use `s.symbols(` to create symbols/variables that allow you to build the mathematical expression for your fitting function
-    """
+    you can then use `s.symbols(` to create symbols/variables that
+    allow you to build the mathematical expression for your fitting
+    function
+    '''
```

Comment:
this is a syntax fix.

note that I'm generating a conflict with the updates to the docstrings.

because these were long run-on lines, I can't easily tell what's going on here -- maybe just keep the suggested update?

Reply:
Are you saying to remove the docstring here that I wrote and keep yours? It is a simple syntax error as you point out -- I have combined the two, but I can remove mine if that is what you are saying.

Reply:
so what I would specifically suggest is:
1) highlight your docstring (shift-V) and do `gq` to get a decent wrapping.
2) compare the docstrings to make sure everything useful from both is included (just do `git diff alec_fitdata pyspecdata/core.py` if you're on this branch, or vice versa

Reply:
I have already merged the branch, but I did precisely this and just commenting here about it for the sake of completeness.